packages/framework/src/Bootstrap.ts

```diff
@@ -79,14 +79,16 @@ export class Bootstrap {
 
   static async createFromPath(bootstrapPath: string = Bootstrap.getBootstrapFilePath()): Promise<Bootstrap> {
     let currentBootstrap = {
-      bootstrap: {},
+      bootstrap: {
+        bootstrapObject: {},
+      },
     };
 
-    if (!bootstrapPath) {
+    if (bootstrapPath) {
```

Comment:
@nmrgt Here is a fix for a typo preventing the existing bootstrap file from being loaded.